src/main/logutils.ts

```diff
@@ -821,16 +619,33 @@ const forceStopRecording = () => {
     const milliSeconds = (videoStopDate.getTime() - videoStartDate.getTime());
     metadata.duration = Math.round(milliSeconds / 1000);
 
+    // If a Keystone Dungeon is in progress, end it properly before we stop recording
+    if (activeChallengeMode) {
+        endChallengeModeDungeon();
+    }
+
     // Clear all kinds of stuff that would prevent the app from starting another
     // recording
     clearCombatants();
 
+    // Regardless of what happens above these lines, _ensure_ that these variables
+    // are cleared.
     activeChallengeMode = undefined;
     testRunning = false;
 
     recorder.stop(metadata, 0);
 }
 
+/**
+ * Watch the logs. Check every second for a new file, 
```

Comment:
Comment has a implementation details that probably don't need to be here.

Reply:
Wasn't me - but I'll removed it 😂